vatri/google-drive-bundle/0.2/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Vatri\\GoogleDriveBundle\\VatriGoogleDriveBundle": ["all"]
```

Comment:
Is this really the correct path?

Reply:
I assume yes. See attached - that's the most recent version of the bundle file.

Also you can check my issue #681. 

![Screenshot_2019-06-29_12-32-47](https://user-images.githubusercontent.com/1230120/60382912-4ec11b80-9a6a-11e9-8158-9f116b542b7e.png)